frontend/src/main/java/dev/totallyspies/spydle/frontend/interface_adapters/view_manager/ViewManagerModel.java

```diff
@@ -36,7 +36,7 @@ public ViewManagerModel(WelcomeView welcomeView, GameRoomView gameRoomView,
 
         // Set up the frame properties
         setTitle("Spydle");
-        setSize(500, 500);
+        setSize(600, 800);
         setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
```

Comment:
I think this is 600, 800 but the rest of the windows you have put is 800, 600
so you should maybe clarify and make sure they are consistent

Reply:
> I think this is 600, 800 but the rest of the windows you have put is 800, 600 so you should maybe clarify and make sure they are consistent. The rest looks amazing!

Yes, I think you are right, I changed it.